src/Components/Key/Key.js

```diff
@@ -27,12 +29,11 @@ class Key extends React.Component {
       base: {
         width: '100%',
         height: 'auto',
-        fontFamily: 'Source Code Pro',
+        fontFamily: MONO_FONT,
         fontWeight: '500',
         fontSize: '13px',
         lineHeight: '13px',
         color: KEY_COLOR,
-        userSelect: 'none',
```

Comment:
When you click to expand/collapse it selects the key (and the text below it), having `userSelect` in place has prevented it.